GPU/TPCSpaceChargeBase/AliTPCSpaceCharge3DCalc.cxx

```diff
@@ -2698,9 +2809,9 @@ void AliTPCSpaceCharge3DCalc::IntegrateDistCorrDriftLineDz(
         (*mDistDPhiRDz)(i, j) = dPhi * radius0;
         (*mDistDz)(i, j) = dzDist;
         /////////////// use irregular grid look up table for correction
-        // set
+        // use oppsite directions of distortion
         (*mCorrIrregularDrDz)(i, j) = -drDist;
-        (*mCorrIrregularDPhiRDz)(i, j) = -dPhi * (radius0 + drDist);
+        (*mCorrIrregularDPhiRDz)(i, j) = -dPhi * radius0;
```

Comment:
Same thing using the distorted points as reference points for the irregular correction lookup tables. Should be 
(*mCorrIrregularDPhiRDz)(i, j) = -1 * dPhi * (radius0 + drDist);